File: Store.API/Store.DOMAIN/Venda/Model/Venda.cs
using Store.DOMAIN.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Store.DOMAIN.Model
{
    public class Venda
    {
        public long IdVenda { get; set; }
        public long IdCliente { get; set; }
        public long IdProduto { get; set; }
        public int QtdVenda { get; set; }
        public double VlrUnitarioVenda { get; set; }
        public DateTime DthVenda { get; set; }
        public double VlrTotalVenda { get; set; }
        public Cliente Cliente { get; set; }
        public Produto Produto { get; set; }
    }
}

File: Store.UI/Store/src/app/domain/venda/venda.ts
import { Cliente } from "../cliente/cliente";
import { Produto } from "../produto/produto";

export class Venda {
  IdVenda : Number = 0;
  IdCliente : Number = 0;
  IdProduto : Number = 0;
  QtdVenda : Number = 0;
  VlrUnitarioVenda : Number = 0;
  DthVenda : Date = new Date();
  VlrTotalVenda : Number = 0;
  Cliente : Cliente = new Cliente();
  Produto : Produto = new Produto();
}
